internal/commands/scan.go

```diff
@@ -384,6 +387,37 @@ func scanShowSubCommand(scansWrapper wrappers.ScansWrapper) *cobra.Command {
 	return showScanCmd
 }
 
+func scanVorpalSubCommand() *cobra.Command {
+	scanVorpalCmd := &cobra.Command{
+		Use:    "vorpal",
+		Hidden: true,
+		Short:  "Run a vorpal scan",
+		Long:   "vorpal scan is able to scan a single file fast and efficiently.",
+		Example: heredoc.Doc(
+			`
+			$ cx scan vorpal --file-source <path to code file> --vorpal-latest-version <use latest Vorpal available version>
+		`,
+		),
+		Annotations: map[string]string{
+			"command:doc": heredoc.Doc(
+				`
+				https://checkmarx.com/resource/documents/en/link-to-cxone-documentation
+			`,
+			),
+		},
+		RunE: runScanVorpalCommand(),
+	}
+
+	scanVorpalCmd.PersistentFlags().Bool(commonParams.VorpalLatestVersion, false, "Set to true to update to latest version")
+	scanVorpalCmd.PersistentFlags().StringP(
+		commonParams.SourcesFlag,
+		commonParams.SourcesFlagSh,
+		"",
+		"The source should be a single file",
```

Comment:
Again,if you are not sure, please comfirm with Eli, what did we wrote in the other commands? let's have the same logic .
Im thinking here about "Source Code File Path"

Reply:
in the other place we use this flag it says 
"Sources like: directory, zip file or git URL."
which is not relevant in this case.
Will take it with Eli.